packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -3747,7 +3795,57 @@ export function attach(
   }
 
   // React will switch between these implementations depending on whether
-  // we have any manually suspended Fibers or not.
+  // we have any manually suspended/errored-out Fibers or not.
+  function shouldErrorFiberAlwaysNull() {
+    return null;
+  }
+
+  const forceErrorForFiberIDs = new Map();
+  function shouldErrorFiberAccordingToMap(fiber) {
+    if (typeof setErrorHandler !== 'function') {
+      throw new Error(
+        'Expected overrideError() to not get called for earlier React versions.',
+      );
+    }
+
+    const id = getOrGenerateFiberID(fiber);
+    let status = null;
+    if (forceErrorForFiberIDs.has(id)) {
+      status = forceErrorForFiberIDs.get(id);
+      if (status === false) {
+        clearErrorsForFiberID(id);
+      }
```

Comment:
This isn't the right place to call this method. `clearErrorsForFiberID` is about console errors (things logged to `console.error`). That should only be cleared when the Fiber unmounts or when the user clicks to clear it.

I don't think we should check for an empty map here either. Conceptually I think this is _read_ method so it should probably not also have side effects. Seems like the `overrideError` method is the better place to add this check (as well as `untrackFibers`).